docs/_extra/api-reference/schemas/group-schema.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "required": [
+    "id",
+    "access",
+    "links",
+    "name",
+    "public",
+    "scoped",
+    "type"
+  ],
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "access": {
+      "type": "object",
+      "required": [
+        "write",
+        "leave"
+      ],
+      "properties": {
+        "write": {
+          "type": "boolean",
+          "description": "true if this user may post annotations to this group"
+        },
+        "leave": {
```

Comment:
Is there a reason it's called 'leave'?

Reply:
It's a URL for leaving the group (ending membership). I'm open to other ideas for naming this property!